services/mailchimp/utilities/index.js

```diff
@@ -78,6 +78,7 @@ export function formatFormInput(event, isEmailEncrypted) {
   return {
     email_address: emailAddress,
     status: 'pending',
+    interests: event.body.interests,
```

Comment:
Does this need a default value by any chance (e.g. `interests: event.body.interests || {}`)?